src/shared/ipc.ts

```diff
@@ -115,6 +115,12 @@ export interface IpcMainTS extends Pick<IpcMain, "listeners"> {
 }
 
 export enum IpcChannel {
-  ApplicationMenuClick = "applicationmenuclick",
-  ContextMenuClick = "contextmenuclick",
+  // Prefix with custom- to avoid naming collisions with built in HTML events.
+  BrowserWindow = "custom-browserwindow",
+  ApplicationMenu = "custom-applicationmenu",
+  ContextMenu = "custom-contextmenu",
```

Comment:
Let's make the event names more specific and prefix them with "ipc". Also we should use prefixed kebab case like this: https://gomakethings.com/custom-event-naming-conventions-in-vanilla-js/

instead of trying to mimic native html events.